test/parallel/test-timers-setInterval-excludes-callback-duration.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+
+const sleep = (ms) => {
+  const st = new Date().getTime();
+  while (new Date().getTime() < st + ms);
+};
+
+let cntr = 0;
+let first, second;
+setInterval(function() {
+  sleep(100);
+  cntr++;
+  if (cntr === 1) {
+    first = new Date();
+  } else if (cntr === 2) {
+    second = new Date();
```

Comment:
In general I would suggest not using wall-clock time, and instead use a monotonic clock by e.g calling `Timer.now()`.